content/content/Advance Usage/gitops.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Gitops"
+date: 2021-11-16T12:38:56-05:00
+draft: false
+---
+
+All the crane commands are individual utilities, but when used together in an order, they form a pipeline. Crane makes it easy to integrate a gitops that applies the patches/resources generated at the end of `apply` on the destination cluster. The resources generated at the end of the process (i.e `export`, `transform`, `apply`), can be pushed to a github repository, and a pipeline can be setup that deploy the resources on a cluster on every push.
+
+One such tool that helps setup this is [argoCD](https://argo-cd.readthedocs.io/en/stable/). [Here](https://argo-cd.readthedocs.io/en/stable/getting_started) is the guide to get started with argoCD and set everything up for gitops pipeline. 
```

Comment:
@eriknelson @sseago Do we want to include steps to configure argoCD also here? I think its better to point to the official doc which includes the steps (description is brief and to the point).

Reply:
Hmm. I'm not sure we're really ready for that here. That might make sense as a follow-on addition to docs once that's better worked out, unless @djzager has some content to add here.

Reply:
I think the first paragraph is fine, we could probably remove the 2nd paragraph and come back to it when we have an actual story to tell.

Reply:
I agree with @sseago and @djzager here. It's on our roadmap to flush this out and when that happens let's circle back and update this.